libraries/ride-fuzzer-lib/pom.xml

```diff
@@ -20,11 +20,6 @@
       <artifactId>ride-core</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
-    <dependency>
-      <groupId>com.github.everit-org.json-schema</groupId>
-      <artifactId>org.everit.json.schema</artifactId>
-      <version>1.9.2</version>
-    </dependency>
```

Comment:
Remving Everite as part of validation because it does not support relative path references in schemas.